plugins/factory-plugin/tests/file-uri.spec.ts

```diff
@@ -26,12 +26,12 @@ describe("Test exec commands", () => {
     });
 });
 
-describe("Testing convertion of project paths to be stored in the workspace config", () => {
-    test("Converting fs project path to che project path", async () => {
-        expect(convertToCheProjectPath('/projects/che-factory-extension/', '/projects')).toBe('/che-factory-extension');
-        expect(convertToCheProjectPath('/projects/che-factory-extension', '/projects')).toBe('/che-factory-extension');
-        expect(convertToCheProjectPath('/projects/che/che-factory-extension/', '/projects')).toBe('/che/che-factory-extension');
-        expect(convertToCheProjectPath('/projects/theiadev_projects/blog.sunix.org/', '/projects/theiadev_projects')).toBe('/blog.sunix.org');
-        expect(convertToCheProjectPath('/projects/che/che-factory-extension/', undefined)).toBe('/che/che-factory-extension');
+describe('Testing convertion of project paths to be stored in the workspace config', () => {
```

Comment:
This is breaking projects cloning from workspace based on `workspace config`. The clonePath is for workspaces based on `devfile`.